src/components/OrgSettings/AgendaItemCategories/Preview/AgendaCategoryPreviewModal.test.tsx

```
@@ -0,0 +1,122 @@
+import { render, screen, fireEvent } from '@testing-library/react';
+import '@testing-library/jest-dom';
+import AgendaCategoryPreviewModal from './AgendaCategoryPreviewModal';
+
+// Mock translation function
+const mockT = (key: string) => key;
+
+describe('AgendaCategoryPreviewModal Component', () => {
+  const mockOnClose = jest.fn();
+  const mockShowUpdateModal = jest.fn();
+  const mockToggleDeleteModal = jest.fn();
+  const mockCategory = { name: 'Meeting', description: 'Team discussion', createdBy: 'John Doe' };
```

ESLint

[error] Replace `·name:·'Meeting',·description:·'Team·discussion',·createdBy:·'John·Doe'` with `⏎····name:·'Meeting',⏎····description:·'Team·discussion',⏎····createdBy:·'John·Doe',⏎·`

(prettier/prettier)